source/docs/articles/sites/enterprise/changemanagement/change-management-for-enterprise.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Change Management for Enterprise Organizations
+description: Learn about the features and benefits of role-based permissions for Pantheon Enterprise Organizations.
+keywords: roles, role, permissions, permission, access, privileges, privilege, team members, developer, organization, enterprise, manage organization, change management, enterprise, one
+---
+Change Management is a feature of Organizations that enables role-based permissions for users in the Organization. Sites which list the Organization as a Supporting Organization are accessible to privileged members of the Organization. The roles exist to restrict who can deploy and manage other users in the Organization or sites it works on.
+
+## Features and Benefits
+* Enterprise Organizations with Change Management can add people as Team Members and Developers to sites.
+* When an Organization is added to a site as a Supporting Organization (e.g. Agencies), the members and roles of the Organization are inherited for working on the site. All Organization members can access all sites associated with the organization.
+* Agencies added to Sites by subscription owners are able to use Change Management internally, but the subscription owners are not—they can only add full team members.
+* All users in an Organization can see other users and their roles.
+
+### Supporting Organizations
+One of the best things about Pantheon is the ability to collaborate with agencies and shops on web projects. If you have contracted with a Pantheon Partner or Ally Agency, you can add them to the site as a _Supporting Organization_, which will give their company access to help build, launch, or maintain your site.  
+
+1. Click **Team** in the upper-right of the Site Dashboard.
+2. Click **Add Supporting Organization** at the bottom of the team management window.
+3. Enter the complete name of the agency. An exact match is required.
+4. Check that the agency is the one you expected.
+5. Click the **Add Agency** button to make them a Supporting Organization.
+
+## Site Roles and Permissions
+From your Org Dashboard, click **Sites**, select a site, then click **Team**. You will see the site owner, team members and their role, and any Supporting Organizations. 
+
+### Team Member
+Team Members are a trusted role that can make changes to the Test and Live environments, add and edit other team members, and add Supporting Organizations.
+
+### Developer
+Developers can only work on the Development environment, and cannot deploy to Test or Live.
```

Comment:
@rachelwhitton Please feel free to add anything you think is useful for Enterprise customers! This is just a starting point and I'm hoping for input.